libcloud/compute/types.py

```diff
@@ -222,25 +222,31 @@ class NodeState(Type):
     Standard states for a node
 
     :cvar RUNNING: Node is running.
+    :cvar STARTING: Node is starting up.
     :cvar REBOOTING: Node is rebooting.
     :cvar TERMINATED: Node is terminated. This node can't be started later on.
+    :cvar STOPPING: Node is currently trying to stop.
```

Comment:
I'm fine with adding new starting and stopping state, but not too sure about reconfiguring.

I know some load balancer as a service offerings expose a similar state, but I'm not aware of too many compute providers. In any case not a big deal.